src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -358,6 +358,18 @@ protected boolean parseArgs(final String[] argv) {
         return true;
     }
 
+    private CommandLineParser determineArgParser(String[] argv) {
+        // If there are args that don't start with dashes and contain equals then we use the old argument parser
+        boolean hasEqualsNoDash = Arrays.stream(argv).anyMatch(arg -> arg.contains("=") && !arg.startsWith("-"));
+
+        commandLineParser = (hasEqualsNoDash) ?
```

Comment:
If the user specifies a different command line flavor than the one inferred, do you want to raise an error? Alternatively, should we ignore the flavor specified and only infer?

Reply:
I think that if the user specifies then no inference should occur.